### PR TITLE
[AR-1537] update images in the careers page

### PR DESCRIPTION
**Pull request by karthik-durai**
# Resolves

- [AR-1537](https://team-1624093970686.atlassian.net/browse/AR-1537)

## Changes

1. Added new photo assets to careers dir.
2. Replaced photo in careers page with photo collage.

# Screenshots
Before:
careers_photo_before

After:
careers_photo_after

# Checklist

- [X] The branch name follows the format: `developer-name/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

### CODE REVIEW

**Review by shrinathprabhu — APPROVED** [review]

File: components/careers/EndSection.vue [hunk 1]
```diff
@@ -31,8 +38,11 @@ export default {
...
 <style lang="postcss" scoped>
-.team-image {
-  width: 100%;
+.team-image-container {
+  max-width: 100%;
   height: 100%;
 }
+.team-image-container:nth-child(2) {
+  margin-top: 100px;
+}
```
Comment (ajithranka) [inline components/careers/EndSection.vue:47]: Can we create a wrapping `div` around the two image rows and set the margin there with a class? The margin value will need to be reduced for small screens.

```
<div class="team-images">
  <v-stack class="team-image-row">...</v-stack>
  <v-stack class="team-image-row">...</v-stack>
</div>
```

```
.team-images {
  margin-top: 100px;

  @media(--viewport-small) {
    margin-top: 50px;
  }
}
```

**Review by ajithranka — CHANGES_REQUESTED** [review]

File: components/careers/EndSection.vue (hunk 1 above)
Comment (karthik-durai) [reply components/careers/EndSection.vue:47]: sure @ajithranka

Comment (karthik-durai) [reply components/careers/EndSection.vue]: @ajithranka done. [commit](https://github.com/arcana-network/website/pull/92/commits/0c4e0055b6605a86b0c150c07fc49841b62c205f)

File: components/careers/EndSection.vue [hunk 2]
```diff
@@ -31,8 +40,17 @@ export default {
...
+@import url('../../components/lib/media-query-helper.css');
+
+.team-images {
+  margin-top: 100px;
+
+  @media (--viewport-small) {
+    margin-top: 50px;
+  }
+}
+.team-image-row {
```
Comment (ajithranka) [inline components/careers/EndSection.vue:52]: [nit] Separate each ruleset by an empty line.

**Review by ajithranka — APPROVED** [review]

**Pull request merged**